src/views/WantHue.vue

```diff
@@ -0,0 +1,858 @@
+          </div>
+        </div>
+        <div class="sort" v-if="bottom === true">
+          Sort by
+          <span @click="paletteColor.sort(SortbyHue)">hue</span>
+          <span @click="paletteColor.sort(SortbyChroma)">chroma</span>
+          <span @click="paletteColor.sort(SortbyLightness)">lightness</span>
+        </div>
+      </div>
+    </div>
+    <div class="bottom" v-if="bottom === true">
+      <div class="colors">
+        <h4>Colors</h4>
+        <div class="color">
+          <div v-for="(item, index) in paletteColor" :key="index">
+            <div class="color-color" :style="{ backgroundColor: item }"></div>
+            <div class="describe">
+              <div class="arg">
+                <div class="number">{{ item }}</div>
+                <div class="key">HEX</div>
+              </div>
+              <div class="arg">
+                <div class="number">{{ chroma(item).rgb().toString() }}</div>
+                <div class="key">RGB</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="json">
+        <h4>JSON</h4>
+        <div>
+          <h5>HEX json</h5>
+          <div class="pre">
+            <div v-for="(item, index) in paletteColor" :key="index">
+              "{{ item }}",
+            </div>
+          </div>
+        </div>
+        <div>
+          <h5>RGB json</h5>
+          <div class="pre">
+            <div v-for="(item, index) in paletteColor" :key="index">
+              {{ chroma(item).rgb() }},
+            </div>
+          </div>
+        </div>
+        <div>
+          <h5>HCL json</h5>
+          <div class="pre">
+            <div v-for="(item, index) in paletteColor" :key="index">
+              {{
+                chroma(item)
+                  .hcl()
+                  .map((item) => item.toFixed(3))
+              }},
+            </div>
+          </div>
+        </div>
+        <div>
+          <h5>LAB json</h5>
+          <div class="pre">
+            <div v-for="(item, index) in paletteColor" :key="index">
+              {{
+                chroma(item)
+                  .lab()
+                  .map((item) => item.toFixed(3))
+              }},
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="json">
+        <h4>CSS</h4>
+        <div>
+          <h5>HEX list for CSS</h5>
+          <div class="pre">
+            <div v-for="(item, index) in paletteColor" :key="index">
+              {{ item }}
+            </div>
+          </div>
+        </div>
+        <div>
+          <h5>RGB list for CSS</h5>
+          <div class="pre">
+            <div v-for="(item, index) in paletteColor" :key="index">
+              rgb({{ chroma(item).rgb().toString() }})
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { onMounted, reactive, ref, watch, watchEffect } from "vue";
```

Comment:
291:43  warning  'watchEffect' is defined but never used  @typescript-eslint/no-unused-vars